pkg/installer/installer.go

```diff
@@ -381,6 +390,19 @@ func (in *Installer) listStorageOSStorageClasses() (*kstoragev1.StorageClassList
 	return stosStorageClassList, nil
 }
 
+func (in *Installer) listStorageOSConfigMaps() (*corev1.ConfigMapList, error) {
+	configMapList, err := pluginutils.ListConfigMaps(in.clientConfig, metav1.ListOptions{LabelSelector: stosAppLabel})
+	if err != nil {
+		return nil, err
+	}
+	stosConfigMapList := &corev1.ConfigMapList{}
+	for _, configMap := range configMapList.Items {
```

Comment:
I don't understand the reason of this loop. `configMapList` is alos a  `corev1.ConfigMapList` why we need to copy config maps here?

Reply:
copy/paste error.. In fact, no need for the extra function at all 😄